practices/practice-two/src/pages/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Practice Two</title>
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link
+      rel="icon"
+      href="https://media.geeksforgeeks.org/wp-content/cdn-uploads/gfg_200X200.png"
+      type="image/x-icon"
+    />
+    <script
```

Comment:
Move at the end of body

Reply:
I need to move the link tag or script tag